docker/ros/kinetic/mongodb-driver/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM shadowrobot/build-tools:xenial-kinetic
+
+LABEL Description="This ROS Kinetic image contains build tools and the required mongoncxx and pyassimp libraries." Vendor="Shadow Robot" Version="1.0"
+
+ENV PROJECTS_WS=/home/user/mongo_ws
+RUN echo "Updating sources" && \
+    apt-get update && \
```

Comment:
if I create the mongo_ws as root then it cannot initialize the folder with the wstool init command